### ISSUE TITLE
Indicate lines to work on - Check before editing

### ISSUE THREAD

**Issue by imcgames** [Treeofsavior/EnglishTranslation#796]
Hi guys, 
since we expect the edits to be made based again from just reading the tsv files after the cbt, 
let's have this issue for indicating which lines you guys are working on so others won't look into the same lines at the same time. 
I'd suggest the range to be in multiples 10's to make it easier? (ex. 550 - 580, not 551 - 589) 

Also, I think it's safe to say at least 1 - 500 if not more than 1000 of most files have been reviewed a few times already as many people tend to start from 0... :sweat:  so maybe those lines can be skipped. 

Btw, with CBT ending, we might be MIA in github for a few days for post-CBT works. :open_mouth: 
I'll still be checking around and stuff but might not be able to merge pull requests during this time. 
I apologize for the expected hiatus but I just wanted to let you guys know so that you wouldn't be so worried for the missing activities. :wink:

**Comment by ttgmichael** [comment]
Lol, gotcha, Sorry for all the weird line starts and ends from my PRs ^^;  It was because I usually started where I find a line that has a problem and go from there xD. 

As you will be on hiatus, should we rather hold off on making PRs then? Because you might be expecting a ton of PRs after hiatus. ^^;

It'll be great to post this on the read me so people will read it! :+1:

**Comment by Sunaries** [comment]
Could you still review the pull requests that were made during the CBT period? 
Also, wasn't this idea on the table before and it didn't work out? @ttgmichael has a valid point concerning creating pull requests during your absence. 

Would you want to come back with over 150 pull requests? (_That could be a slight exaggeration_) :stuck_out_tongue_winking_eye:

**Comment by Nagoda** [comment]
I've become strongly convinced that the "Attribute" passive skill system needs to be renamed, and I believe that replacing it with the term "Talents" is the best choice. **I intend to start this process by filing a pull request for the Skills.tsv file that will contain a very large number of find>replace operations.**

I'm not a Korean speaker, but my experience with the game through both the CBT, and reviewing the .tsv text files had made it apparent to me that the name itself is troublesome due to its similarity to other frequently used terms within the game, and issues with inconsistent translations in the current text. 

_See my most recent two posts in the discussion here for more of my reasoning:_ https://github.com/Treeofsavior/EnglishTranslation/issues/731

I intend to summarize and restate this reasoning in the comments of my pull request.

**What this means for other contributors, is that you may want to hold off for a few days on submitting pull requests for the Skills.tsv file that affect any line containing the word "attribute"** (Specifically, any "attribute" that appears on line 2005 to the end of the file)

I intend to make this pull request once IMC has cleared out all the current PRs that have stacked up over the CBT period. Until that time, I'm receptive to hearing the arguments of others if they believe that the current naming scheme needs to stay for some reason, or someone believes they have a better idea. If not, IMC will be able to review the pull request and they can make a final decision on whether they agree.

If this merge goes through, some additional cleanup will be required in the UI and ETC files, and any NPC dialogue that refers to "Attributes", such as class masters.

**Comment by ttgmichael** [comment]
@Nagoda's post reminded me that we still need to straighten out standardized terms: stuff like Attributes/ Statuses/ Blunts/Pierce/ Quarrelshooter/ ea or x2/ etc.

Let's hold off from any PRs during this hiatus to discuss this and straighten out things. When IMC comes back maybe we can get the wiki really updated ^^.

**Comment by vtange** [comment]
<b>Most Importantly</b>, I think we need to quickly make a system that marks what lines were contributed as a real time translation in-game (and thus should be a very accurate/usable line). This way we won't have people coming in and erasing our hard work during this last week.

**Comment by laurencee** [comment]
@Nagoda I agree that "Attribute" has caused nothing but confusion when talking with various people about the game. Talents/Passives/Mods are all better terms which fit better with existing games.

Stats/Attributes have been almost exclusively used to represent the primary STR/INT/DEX etc. values in english RPGs.

**Comment by emailboxu** [comment]
Personally I'm a fan of the term "Passives", it's a common term for the kinds of upgrades Attributes are.

**Comment by laurencee** [comment]
@emailboxu That's my preferred term too.

**Comment by Sunaries** [comment]
"Passives" :+1:  

P.S. 
_(Is anyone up to join a [Discord](https://discordapp.com/) server that I could create to discuss the translations of TOS? This way you can participate on Github, the forums, but also on Discord with fellow translators. )_

**What is Discord?**
Discord is a new VoIP app that is free to use. There are no costs for servers, accounts, etc. You can also use Discord from mobile apps, desktop apps, or just through the browser.​ It is a very good alternative to Skype. (_It doesn't hog your CPU as Skype does..._)

**Comment by Ragunaga** [comment]
I'd love a discord channel Sunaries.
It'd be a great help for all sorts of things.

**Comment by Sunaries** [comment]
> I'd love a discord channel Sunaries.
> It'd be a great help for all sorts of things.

:smile:  I set up a server. Contributors are welcome to join. [Click here to join the Discord server.](https://discord.gg/0RoYSiES4f4zwAk8)

**Comment by emailboxu** [comment]
I'll join too, once I get home. My flight was delayed :(
